lita-locker.gemspec

```diff
@@ -16,6 +16,7 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'lita', '>= 4.0.1'
   spec.add_runtime_dependency 'redis-objects'
+  spec.add_runtime_dependency 'actionview'
```

Comment:
Aww c'mon, doesn't everyone want to install nokogiri?  :laughing:  Updated to use time-lord.